fetch/api/response/response-static-json.any.js

```diff
@@ -0,0 +1,64 @@
+// META: global=window,worker
+// META: title=Response: json static method
+
+const APPLICATION_JSON = "application/json;charset=utf-8";
+const FOO_BAR = "foo/bar";
+
+const DEFAULT_HEADERS = { "content-type": "application/json;charset=utf-8" };
+const CUSTOM_HEADERS = { "content-type": "foo/bar" };
+const CUSTOM_HEADERS = { "x-foo": "bar" };
```

Comment:
one of these needs to be renamed, or all 3 should be removed as theyre not being used